client/components/header/header.js

```
@@ -61,37 +61,110 @@ export default class Header extends React.Component {
-    const {
-      header,
-      languages,
-      language,
-      orgSlug,
-      setLanguage,
-      location,
-      isAuthenticated,
-      userData,
-    } = this.props;
-    const {logo, links, second_logo: secondLogo} = header;
+  getLogoMarkup = (logo, orgSlug, imageClassName) =>
```

ESLint

[error] Expected 'this' to be used by class method 'getLogoMarkup'.

(class-methods-use-this)